### ISSUE TITLE
Unread Count adds one when reading an email, Subtracts when Exiting Email

### ISSUE THREAD

**Issue by apersonnamedmike** [Thomas101/wmail#510]
* Which version of WMail are you using? 2.0.9 (Prerelease Channel)

* Which Operating System are you using? Windows 10

(Using Google Inbox)
When Scraping unread count directly from ui, and click on an already read email to read through again, it'll say that an unread email is in the inbox. When clicking out of the email, it puts the unread count back to what it really is.

**Comment by Thomas101** [comment]
Thanks! I've noticed this too. I think it's down to this patch (https://github.com/Thomas101/wmail/commit/c82a2481eff0c7c85b7cfac9bc97930c1b2c8b70) which fixes an issue when you expand a bundle/cluster. I'll take another look

**Comment by apersonnamedmike** [comment]
Sounds good! Thanks :)

**Comment by Thomas101** [comment]
Hi, I've removed the DOM scraping from the 2.2.0 and 2.2.1 release as it's not robust enough. I have been able to improve the query that I run on the API though. It's enabled by default on Inbox accounts, however on previously added accounts you'll need to change the setting. If you head into settings and then your account. Under `unread mode` select `All Unread Unbundled messages`. The new versions are available to [download here](https://thomas101.github.io/wmail/download?v=2.2.*). Thanks!
